_scripts/mysql.sh

```diff
@@ -1,12 +1,26 @@
 #!/bin/bash -ex
-container_id=$(docker ps -a | grep mydb | cut -d' ' -f1)
 
-if [ -n "$container_id" ]; then
+# This docker image doesn't react to SIGINTs (Ctrl+C) which is used by
+# pm2 to kill processes.
+# We go around this by defining a SIGINT handler, running docker in the
+# background (but still logging on the screen), and running a read to keep
+# the script running.
+
+function on_singint() {
+  echo "MySQL shutting down."
   docker stop mydb
-fi
+  exit 0
+}
+
+trap on_singint INT
 
+# Create pushbox db on start (because pushbox doesn't create it)
 docker run --rm --name=mydb \
--e MYSQL_ALLOW_EMPTY_PASSWORD=true \
--e MYSQL_ROOT_HOST=% \
--p 3306:3306 \
-mysql/mysql-server:5.6
+  --network fxa-net \
+  -e MYSQL_ALLOW_EMPTY_PASSWORD=true \
+  -e MYSQL_ROOT_HOST=% \
+  -e MYSQL_DATABASE=pushbox \
+  -p 3306:3306 \
+  mysql/mysql-server:5.6 &
```

Comment:
This still references mysql-server:5.6, but the diff about only pulls 5.7; do we need to pull both?